homeassistant/scripts/benchmark/__init__.py

```diff
@@ -23,6 +23,10 @@ def run(args):
         description=("Run a Home Assistant benchmark."))
     parser.add_argument('name', choices=BENCHMARKS)
     parser.add_argument('--script', choices=['benchmark'])
+    parser.add_argument(
```

Comment:
Instead of adding it to each script, let's create a function to create a "base" `ArgumentParser`.

```python
def create_script_argparse(description, script_name):
    parser = …
    parser.add_argument('--skip-pip' …)
    return parser
```

Reply:
What's the `description` meant for?

Reply:
I think we will need to add a function to each script, e.g. called `add_args()`. This will allow to add the arguments before running the script which is necessary because the args need to be parsed before running in order to find out if `--skip-pip` is set.

Reply:
So, are you okay with `add_args()`?

Reply:
The description is because it's needed to initialize the object. I'm fine with an add args function too.